# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
#
# Translators:
msgid ""
msgstr ""
"Project-Id-Version: Spacewalk\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2020-09-29 14:21+0200\n"
"PO-Revision-Date: 2020-09-12 19:48+0000\n"
"Last-Translator: Kukuh Syafaat <syafaatkukuh@gmail.com>\n"
"Language-Team: Indonesian <https://l10n.opensuse.org/projects/uyuni/backend/"
"id/>\n"
"Language: id\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"X-Generator: Weblate 3.6.1\n"

#: ../server/apacheAuth.py:66
#, fuzzy, python-format
msgid "incomplete proxy authentication token: %s"
msgstr "token otentikasi proksi tidak lengkap: %s"

#: ../server/apacheAuth.py:84
#, fuzzy, python-format
msgid "Sent proxy signature %s does not match ours %s."
msgstr "Tanda tangan proksi terkirim %s tidak cocok dengan %s kita."

#: ../server/apacheAuth.py:119
#, fuzzy
msgid "declined proxy authentication"
msgstr "otentikasi proxy yang ditolak"

#: ../server/handlers/app/packages.py:192
#: ../server/handlers/xmlrpc/registration.py:215
#: ../server/handlers/xmlrpc/registration.py:1168
#: ../server/handlers/xmlrpc/registration.py:1307
#: ../server/configFilesHandler.py:129
#, fuzzy, python-format
msgid ""
"\n"
"                   %s Account has been deactivated on this server.\n"
"                   Please contact your Org administrator for more help."
msgstr ""
"Akun %s telah dinonaktifkan pada server ini.\n"
"                   Silakan hubungi administrator Organisasi Anda untuk "
"bantuan lebih lanjut."

#: ../server/handlers/applet/applet.py:78
#, fuzzy, python-brace-format
msgid "Your system was not found in the {PRODUCT_NAME} database"
msgstr "Sistem Anda tidak ditemukan dalam rhn database"

#: ../server/handlers/sat/auth.py:45
#, fuzzy
msgid "ISS is disabled on this server."
msgstr "ISS dinonaktifkan di server ini."

#: ../server/handlers/sat/auth.py:50
#, fuzzy, python-format
msgid "Server \"%s\" is not enabled for ISS."
msgstr "Server \"%s\" tidak diaktifkan untuk ISS."

#: ../server/handlers/xmlrpc/scap.py:104
#, fuzzy
msgid "Invalid system_id/action_id pair."
msgstr "Pasangan system_id/action_id tidak valid."

#: ../server/handlers/xmlrpc/errata.py:64
#, fuzzy, python-format
msgid "Expected a package name, not: %s"
msgstr "Diduga sebuah nama paket, bukan: %s"

#: ../server/handlers/xmlrpc/errata.py:115
#, fuzzy, python-format
msgid "Expected a package, not: %s"
msgstr "Diduga sebuah paket, bukan: %s"

#: ../server/handlers/xmlrpc/queue.py:433
#, fuzzy, python-format
msgid "Invalid action value type %s (%s)"
msgstr "Tipe nilai tindakan %s (%s) tidak valid"

#: ../server/handlers/xmlrpc/queue.py:464
#, fuzzy, python-format
msgid "Action %s does not belong to server %s"
msgstr "Tindakan %s bukan milik server %s"

#: ../server/handlers/xmlrpc/up2date.py:233
#, fuzzy, python-format
msgid "Invalid value %s (%s)"
msgstr "Nilai %s (%s) tidak valid"

#: ../server/handlers/xmlrpc/proxy.py:65
#, fuzzy, python-format
msgid "SUSE Manager Proxy service not enabled for server profile: \"%s\""
msgstr ""
"Layanan Proksi Manajer SUSE tidak diaktifkan untuk profil server: \"%s\""

#: ../server/handlers/xmlrpc/proxy.py:85
#, fuzzy
msgid "Invalid session key"
msgstr "Kunci sesi tidak valid"

#: ../server/handlers/xmlrpc/proxy.py:93
#, fuzzy
msgid "This server ID no longer exists"
msgstr "ID server ini tidak ada lagi"

#: ../server/handlers/xmlrpc/registration.py:194
#, fuzzy
msgid "Required data missing"
msgstr "Data yang diperlukan hilang"

#: ../server/handlers/xmlrpc/registration.py:198
#, fuzzy
msgid "Required members missing"
msgstr "Anggota yang diperlukan hilang"

#: ../server/handlers/xmlrpc/registration.py:207
#, fuzzy
msgid "Attempt to register a system to an invalid username"
msgstr "Mencoba mendaftarkan sistem ke nama pengguna yang tidak valid"

#: ../server/handlers/xmlrpc/registration.py:839
#, fuzzy
msgid "Invalid system release version requested"
msgstr "Versi rilis sistem tidak valid yang diminta"

#: ../server/handlers/xmlrpc/registration.py:1063
#, fuzzy
msgid ""
"Unable to find a valid network interface, both ipaddr and ip6addr not found."
msgstr ""
"Tidak dapat menemukan antarmuka jaringan yang valid, ipaddr dan ip6addr "
"tidak ditemukan."

#: ../server/handlers/xmlrpc/registration.py:1115
#, python-brace-format
msgid "{PRODUCT_NAME} Welcome Message"
msgstr ""

#: ../server/handlers/xmlrpc/registration.py:1125
#, fuzzy
msgid "Privacy Statement"
msgstr "Red Hat Pernyataan Privasi Satelit"

#: ../server/handlers/xmlrpc/registration.py:1146
#, fuzzy
msgid "Expected a dictionary as a product argument"
msgstr "Mengharapkan kamus sebagai argumen produk"

#: ../server/handlers/xmlrpc/registration.py:1374
#, fuzzy, python-format
msgid "Invalid value '%s' for %s (%s)"
msgstr "Nilai '%s' tidak valid untuk %s (%s)"

#: ../server/importlib/userAuth.py:98
#, fuzzy
msgid "You are not authorized to manage packages in the null org"
msgstr "Anda tidak berwenang untuk mengelola paket di null org"

#: ../server/importlib/userAuth.py:103
#, fuzzy, python-format
msgid "You are not allowed to manage packages in the %s org"
msgstr "Anda tidak diizinkan untuk mengelola paket dalam organisasi %s"

#: ../server/importlib/userAuth.py:121
#, fuzzy
msgid "You are not allowed to perform administrative tasks"
msgstr "Anda tidak diizinkan untuk melakukan tugas administratif"

#: ../server/importlib/userAuth.py:144
#, fuzzy, python-format
msgid ""
"You are not allowed to manage channel %s, or that channel does not exist"
msgstr ""
"Anda tidak diizinkan untuk mengelola saluran %s, atau saluran tersebut tidak "
"ada"

#: ../server/importlib/userAuth.py:204
#, fuzzy, python-format
msgid ""
"\n"
"               %s Account has been deactivated on this server.\n"
"               Please contact your Org administrator for more help."
msgstr ""
"Akun %s telah dinonaktifkan pada server ini.\n"
"               Silakan hubungi administrator Organisasi Anda untuk bantuan "
"lebih lanjut."

#: ../server/rhnHandler.py:62
#, fuzzy
msgid "Please run rhn_register as root on this client"
msgstr "Silakan jalankan rhn_register sebagai root pada klien ini"

#: ../server/rhnHandler.py:75
#, fuzzy, python-format
msgid "Service not enabled for system profile: \"%s\""
msgstr "Layanan tidak diaktifkan untuk profil sistem: \"%s\""

#: ../server/rhnServer/server_class.py:97
#, fuzzy, python-format
msgid "Architecture `%s' is not supported"
msgstr "Arsitektur '%s' tidak didukung"

#: ../server/rhnServer/server_token.py:78
#, fuzzy, python-format
msgid "Token `%s' has more than one base channel assigned"
msgstr "Token '%s' memiliki lebih dari satu saluran dasar yang ditugaskan"

#: ../server/rhnServer/server_token.py:80
#, fuzzy
msgid "Conflicting base channels"
msgstr "Saluran dasar yang bertentangan"

#: ../server/rhnServer/server_token.py:190
#, fuzzy, python-format
msgid "Failed to add server to group %s"
msgstr "Gagal menambahkan server ke grup %s"

#: ../server/rhnServer/server_token.py:503
#, fuzzy, python-format
msgid "Maximum usage count of %s reached"
msgstr "Jumlah penggunaan maksimum dari %s tercapai"

#: ../server/rhnServer/server_token.py:744
#, fuzzy
msgid ""
"Stacking of re-registration tokens with different base entitlements is not "
"supported"
msgstr ""
"Penumpukan token pendaftaran ulang dengan hak dasar yang berbeda tidak "
"didukung"

#: ../server/rhnServer/server_token.py:811
#, fuzzy, python-format
msgid "Could not find token '%s'"
msgstr "Tidak dapat menemukan token '%s'"

#. Cannot use activation keys from different orgs
#: ../server/rhnServer/server_token.py:843
#, fuzzy
msgid "Tokens from mismatching orgs"
msgstr "Token dari organisasi yang tidak cocok"

#. Two tokens with different kickstart sessions
#: ../server/rhnServer/server_token.py:853
#, fuzzy
msgid "Kickstart session mismatch"
msgstr "Ketidakcocokan sesi kickstart"

#: ../server/rhnServer/server_token.py:877
#, fuzzy
msgid "Stacking of re-registration tokens is not supported"
msgstr "Penumpukan token pendaftaran ulang tidak didukung"

#: ../server/rhnUser.py:289 ../server/rhnUser.py:292
#, fuzzy
msgid "Invalid username/password combination"
msgstr "Kombinasi nama pengguna/kata sandi tidak valid"

#. nope, not reserved either
#: ../server/rhnUser.py:467
#, fuzzy, python-format
msgid "Username `%s' has not been reserved"
msgstr "Nama pengguna '%s' belum dicadangkan"

#. New accounts have to specify an e-mail address
#: ../server/rhnUser.py:473
#, fuzzy
msgid "E-mail address not specified"
msgstr "Alamat e-mail tidak ditentukan"

#: ../server/rhnUser.py:509
#, fuzzy, python-format
msgid "username should be at least %d characters"
msgstr "nama pengguna harus setidaknya %d karakter"

#: ../server/rhnUser.py:512
#, fuzzy, python-format
msgid "username should be less than %d characters"
msgstr "nama pengguna harus kurang dari %d karakter"

#: ../server/rhnUser.py:523
#, fuzzy, python-format
msgid "username = `%s', invalid character `%s'"
msgstr "username = '%s', karakter '%s' tidak valid"

#: ../server/rhnUser.py:542
#, fuzzy, python-format
msgid "Please limit your e-mail address to %s chars"
msgstr "Batasi alamat e-mail Anda ke %s chars"

#: ../server/rhnUser.py:623
#, fuzzy, python-format
msgid "password must be at least %d characters"
msgstr "kata sandi harus setidaknya %d karakter"

#: ../server/rhnUser.py:626
#, fuzzy, python-format
msgid "Password must be shorter than %d characters"
msgstr "Kata sandi harus lebih pendek dari karakter %d"

#: ../server/rhnUser.py:644
#, fuzzy, python-format
msgid "password contains character `%s'"
msgstr "kata sandi berisi karakter '%s'"

#: ../server/rhnUser.py:651
#, fuzzy, python-format
msgid "username should be at least %d characters long"
msgstr "username harus setidaknya %d karakter panjang"

#: ../server/rhnUser.py:662
#, fuzzy, python-format
msgid "Cannot register usernames ending with %s"
msgstr "Tidak dapat mendaftarkan nama pengguna yang diakhiri dengan %s"

#: ../server/rhnPackage.py:49 ../server/rhnPackage.py:87
#: ../server/rhnPackage.py:107 ../server/rhnPackage.py:158
#, fuzzy, python-format
msgid "Invalid RPM package %s requested"
msgstr "Paket RPM %s tidak valid diminta"

#: ../server/rhnPackage.py:112
#, fuzzy
msgid "Package not found"
msgstr "Paket tidak ditemukan"

#: ../server/apacheRequest.py:59
#, fuzzy, python-format
msgid "Invalid request received (%s)."
msgstr "Permintaan tidak valid diterima (%s)."

#: ../server/apacheRequest.py:120
#, fuzzy, python-format
msgid ""
"IMPORTANT MESSAGE FOLLOWS:\n"
"%s"
msgstr ""
"PESAN PENTING BERIKUT:\n"
"%s"

#: ../server/apacheRequest.py:149
#, fuzzy, python-format
msgid ""
"While running '%s': caught\n"
"%s : %s\n"
msgstr ""
"Saat menjalankan '%s': tertangkap\n"
"%s : %s (Dalam Bahasa Inggris)"

#: ../server/apacheRequest.py:606
#, fuzzy, python-format
msgid "Invalid URI %s"
msgstr "URI “%s” Tidak Sah"

#: ../server/rhnChannel.py:1992
#, fuzzy, python-format
msgid ""
"Your account does not have access to any channels matching "
"(release='%(release)s', arch='%(arch)s')%(www_activation)s"
msgstr ""
"Akun Anda tidak memiliki akses ke saluran apa pun yang cocok "
"(release='%(release)s', arch='%(arch)s')%(www_activation)s"

#: ../server/rhnChannel.py:2002
#, fuzzy
msgid ""
"\n"
"If you have a registration number, please register with it first at http://"
"www.redhat.com/apps/activate/ and then try again.\n"
"\n"
msgstr ""
"Jika Anda memiliki nomor registrasi, silakan mendaftar dengan terlebih "
"dahulu di http://www.redhat.com/apps/activate/ dan kemudian coba lagi."

#: ../server/rhnChannel.py:2012
#, fuzzy, python-format
msgid "Insufficient subscription permissions for release (%s, %s"
msgstr "Izin berlangganan tidak cukup untuk rilis (%s, %s"

#. Various messages that can be reused
#.
#. bretm 02/07/2007 -- when we have better old-client documentation, probably
#. will be safe to get rid of all this crap
#: ../server/rhnChannel.py:2276 ../server/rhnChannel.py:2287
#: ../server/rhnChannel.py:2302
#, fuzzy
msgid "System Registered but Inactive"
msgstr "Sistem Terdaftar tetapi Tidak Aktif"

#: ../server/rhnChannel.py:2277
#, fuzzy, python-format
msgid ""
"\n"
"Invalid Architecture and OS release combination (%s, %s).\n"
"Your system has been registered, but will not receive updates\n"
"because it is not subscribed to a channel. If you have not yet\n"
"activated your product for service, please visit our website at:\n"
"\n"
"     http://www.redhat.com/apps/activate/\n"
"\n"
"...to activate your product."
msgstr ""
"Kombinasi pelepas Arsitektur dan OS tidak valid (%s, %s).\n"
"Sistem Anda telah terdaftar, tetapi tidak akan menerima pembaruan\n"
"karena tidak berlangganan saluran. Jika Anda belum\n"
"mengaktifkan produk Anda untuk layanan, silakan kunjungi website kami di:\n"
"\n"
"http://www.redhat.com/apps/activate/\n"
"\n"
"... untuk mengaktifkan produk Anda."

#: ../server/rhnChannel.py:2288
#, fuzzy, python-format
msgid ""
"\n"
"Invalid Architecture and OS release combination (%s, %s).\n"
"Your system has been registered, but will not receive updates\n"
"because it could not be subscribed to a base channel.\n"
"Please contact your organization administrator for assistance.\n"
msgstr ""
"Kombinasi pelepas Arsitektur dan OS tidak valid (%s, %s).\n"
"Sistem Anda telah terdaftar, tetapi tidak akan menerima pembaruan\n"
"karena tidak dapat di subscribe ke base channel.\n"
"Hubungi administrator organisasi Anda untuk mendapatkan bantuan."

#: ../server/rhnChannel.py:2295
#, fuzzy, python-format
msgid ""
"\n"
"  This system has been successfully registered, but is not yet entitled\n"
"  to service.  To entitle this system to service, login to the web site at:\n"
"\n"
"  %(entitlement_url)s\n"
msgstr ""
"Sistem ini telah berhasil didaftarkan, namun belum\n"
"  layanan.  Untuk memberikan hak sistem ini ke layanan, login ke situs web "
"di:\n"
"\n"
"%(entitlement_url)s (Dalam Bahasa Inggris)"

#: ../server/rhnChannel.py:2303
#, fuzzy, python-format
msgid ""
"\n"
"  This system has been successfully registered, but no service entitlements\n"
"  were available.  To entitle this system to service, login to the web site "
"at:\n"
"\n"
"  %(entitlement_url)s\n"
msgstr ""
"Sistem ini telah berhasil didaftarkan, tetapi tidak ada hak layanan\n"
"  tersedia.  Untuk memberikan hak sistem ini ke layanan, login ke situs web "
"di:\n"
"\n"
"%(entitlement_url)s (Dalam Bahasa Inggris)"

#. 0-999: Red Hat client/client-like interaction errors:
#: ../common/rhnException.py:53
#, fuzzy
msgid "This does not appear to be a valid username."
msgstr "Ini tampaknya tidak menjadi nama pengguna yang valid."

#: ../common/rhnException.py:54
#, fuzzy
msgid "Invalid username and password combination."
msgstr "Kombinasi nama pengguna dan kata sandi tidak valid."

#: ../common/rhnException.py:55
#, fuzzy
msgid "This login is already taken, or the password is incorrect."
msgstr "Login ini sudah diambil, atau sandi tidak benar."

#: ../common/rhnException.py:56
#, fuzzy
msgid "Permission denied."
msgstr "Hak akses ditolak."

#: ../common/rhnException.py:57
#, fuzzy
msgid "Object not found."
msgstr "Objek tidak ditemukan"

#: ../common/rhnException.py:58
#, fuzzy
msgid "Invalid System Digital ID."
msgstr "ID Digital sistem tidak valid."

#: ../common/rhnException.py:59
#, fuzzy
msgid "Invalid System Credentials."
msgstr "Kredensial sistem tidak valid."

#: ../common/rhnException.py:60
#, fuzzy
msgid "Could not retrieve user data from database."
msgstr "Tak bisa mengambil data pengguna dari pangkalan data."

#: ../common/rhnException.py:61
#, fuzzy
msgid "Valid username required."
msgstr "Diperlukan nama pengguna yang valid."

#: ../common/rhnException.py:62
#, fuzzy
msgid "Valid password required."
msgstr "Diperlukan kata sandi yang valid."

#: ../common/rhnException.py:63
#, fuzzy
msgid "Minimum username length violation."
msgstr "Pelanggaran panjang nama pengguna minimum."

#: ../common/rhnException.py:64
#, fuzzy
msgid "Minimum password length violation."
msgstr "Pelanggaran panjang kata sandi minimum."

#: ../common/rhnException.py:65
#, fuzzy
msgid "The username contains invalid characters."
msgstr "Nama pengguna berisi karakter yang tidak valid."

#: ../common/rhnException.py:66
#, fuzzy
msgid "File not found."
msgstr "Berkas tidak ditemukan."

#: ../common/rhnException.py:67
#, fuzzy
msgid "Architecture and OS version combination is not supported."
msgstr "Kombinasi versi arsitektur dan OS tidak didukung."

#: ../common/rhnException.py:68
#, fuzzy
msgid "Could not retrieve system data from database."
msgstr "Tidak dapat mengambil data sistem dari database."

#: ../common/rhnException.py:69
#, fuzzy
msgid "Invalid arguments passed to function."
msgstr "Argumen tidak valid diteruskan ke fungsi."

#: ../common/rhnException.py:70
#, fuzzy
msgid "Unable to retrieve requested entry."
msgstr "Tidak dapat mengambil entri yang diminta."

#: ../common/rhnException.py:71
#, fuzzy
msgid "Could not update database entry."
msgstr "Tidak dapat memperbarui entri database."

#: ../common/rhnException.py:72
#, fuzzy
msgid "Unsupported server architecture."
msgstr "Arsitektur server yang tidak didukung."

#: ../common/rhnException.py:73
#, fuzzy, python-format
msgid ""
"\n"
"     The anonymous server functionality is no longer available.\n"
"\n"
"     Please re-register this system by running mgr_register as root.\n"
"     Please visit https://%(hostname)s/rhn/systems/SystemEntitlements.do\n"
"     or login at https://%(hostname)s, and from the \"Overview\" tab,\n"
"     select \"Subscription Management\" to enable the service for this "
"system.\n"
"     "
msgstr ""
"Fungsionalitas server anonim tidak lagi tersedia.\n"
"\n"
"Silakan mendaftar ulang sistem ini dengan menjalankan mgr_register sebagai "
"root.\n"
"     Silakan kunjungi https://%(hostname)s/rhn/systems/SystemEntitlements."
"do\n"
"     atau login di https://%(hostname)s, dan dari tab \"Ikhtisar\",\n"
"     pilih \"Manajemen Langganan\" untuk mengaktifkan layanan untuk sistem "
"ini."

#: ../common/rhnException.py:81
#, fuzzy
msgid "Record not available in the database."
msgstr "Catatan tidak tersedia dalam database."

#: ../common/rhnException.py:82
#, fuzzy
msgid "Invalid value for entry."
msgstr "Nilai tidak valid untuk entri."

#: ../common/rhnException.py:83
#, fuzzy, python-format
msgid ""
"\n"
"     This system does not have a valid entitlement for SUSE Manager.\n"
"     Please visit https://%(hostname)s/rhn/systems/SystemEntitlements.do\n"
"     or login at https://%(hostname)s, and from the \"Overview\" tab,\n"
"     select \"Subscription Management\" to enable the service for this "
"system.\n"
"     "
msgstr ""
"Sistem ini tidak memiliki hak yang valid untuk Manajer SUSE.\n"
"     Silakan kunjungi https://%(hostname)s/rhn/systems/SystemEntitlements."
"do\n"
"     atau login di https://%(hostname)s, dan dari tab \"Ikhtisar\",\n"
"     pilih \"Manajemen Langganan\" untuk mengaktifkan layanan untuk sistem "
"ini."

#: ../common/rhnException.py:89
#, fuzzy
msgid "Channel error"
msgstr "Kesalahan saluran"

#: ../common/rhnException.py:90
#, fuzzy
msgid "Client session token is invalid."
msgstr "Token sesi klien tidak valid."

#: ../common/rhnException.py:91
#, fuzzy
msgid "Client session token has expired."
msgstr "Token sesi klien telah kedaluwarsa."

#: ../common/rhnException.py:92
#, fuzzy
msgid "You are not authorized to retrieve the requested object."
msgstr "Anda tidak berwenang untuk mengambil objek yang diminta."

#: ../common/rhnException.py:93
#, fuzzy
msgid "Invalid action"
msgstr "Tindakan tidak sah"

#: ../common/rhnException.py:94
#, fuzzy
msgid "You are not allowed to perform administrative tasks on this system."
msgstr ""
"Anda tidak diizinkan untuk melakukan tugas administratif pada sistem ini."

#: ../common/rhnException.py:96
#, fuzzy
msgid "The system is already subscribed to the specified channel."
msgstr "Sistem ini sudah berlangganan ke saluran yang ditentukan."

#: ../common/rhnException.py:97
#, fuzzy
msgid "The system is not currently subscribed to the specified channel."
msgstr "Sistem saat ini tidak berlangganan saluran yang ditentukan."

#: ../common/rhnException.py:98
#, fuzzy
msgid "The specified channel does not exist."
msgstr "Saluran yang ditentukan tidak ada."

#: ../common/rhnException.py:99
#, fuzzy
msgid "Invalid channel version."
msgstr "Versi saluran tidak valid."

#: ../common/rhnException.py:100
#, fuzzy
msgid ""
"\n"
"     User group membership limits exceeded.\n"
"\n"
"     The current settings for your account do not allow you to add another\n"
"     user account. Please check with the organization administrator for "
"your\n"
"     account if the maximum number of users allowed to subscribe to server "
"needs\n"
"     to be changed.\n"
"     "
msgstr ""
"Batas keanggotaan grup pengguna terlampaui.\n"
"\n"
"Pengaturan saat ini untuk akun Anda tidak memungkinkan Anda menambahkan\n"
"     akun pengguna. Silakan hubungi administrator organisasi untuk\n"
"     akun jika jumlah maksimum pengguna yang diizinkan untuk berlangganan "
"kebutuhan server\n"
"     untuk diubah."

#: ../common/rhnException.py:108
#, fuzzy
msgid ""
"\n"
"     System group membership limits exceeded.\n"
"\n"
"     The current settings for your account do not allow you to add another\n"
"     system profile. Please check with the organization administrator for "
"your\n"
"     account for modifying the maximum number of system profiles that can "
"be\n"
"     subscribed to your account.\n"
"     "
msgstr ""
"Batas keanggotaan grup sistem terlampaui.\n"
"\n"
"Pengaturan saat ini untuk akun Anda tidak memungkinkan Anda menambahkan\n"
"     profil sistem. Silakan hubungi administrator organisasi untuk\n"
"     akun untuk memodifikasi jumlah maksimum profil sistem yang dapat\n"
"     berlangganan akun Anda."

#: ../common/rhnException.py:116
#, fuzzy
msgid ""
"\n"
"     Invalid architecture.\n"
"\n"
"     The architecture of the package is not supported by\n"
"     "
msgstr ""
"Arsitektur tidak valid.\n"
"\n"
"Arsitektur paket tidak didukung oleh"

#: ../common/rhnException.py:121
#, fuzzy
msgid "Invalid RPM header"
msgstr "Header RPM tidak valid"

#: ../common/rhnException.py:122
#, fuzzy
msgid ""
"\n"
"    This system is already registered as a Salt Minion. If you want to "
"register it as a traditional client\n"
"    please delete it first via the web UI or API and then register it using "
"the traditional tools.\n"
"    "
msgstr ""
"Sistem ini sudah terdaftar sebagai Salt Minion. Jika Anda ingin "
"mendaftarkannya sebagai klien tradisional\n"
"    silakan hapus terlebih dahulu melalui UI web atau API dan kemudian "
"daftarkan menggunakan alat tradisional."

#. For the uploading tools
#: ../common/rhnException.py:127
#, fuzzy
msgid "Invalid information uploaded to the server"
msgstr "Informasi tidak valid yang diunggah ke server"

#: ../common/rhnException.py:128
#, fuzzy
msgid "Error uploading network interfaces configuration."
msgstr "Galat saat mengunggah konfigurasi antarmuka jaringan."

#: ../common/rhnException.py:129
#, fuzzy
msgid ""
"\n"
"     Package Upload Failed due to uniqueness constraint violation.\n"
"     Make sure the package does not have any duplicate dependencies or\n"
"     does not already exists on the server\n"
"     "
msgstr ""
"Upload Paket Gagal karena pelanggaran kendala keunikan.\n"
"     Pastikan paket tidak memiliki dependensi duplikat atau\n"
"     belum ada di server"

#: ../common/rhnException.py:134
#, fuzzy
msgid ""
"\n"
"     The --force rhnpush option is disabled on this server.\n"
"     Please contact your SUSE Manager administrator for more help.\n"
"     "
msgstr ""
"Opsi --force rhnpush dinonaktifkan di server ini.\n"
"     Hubungi administrator SUSE Manager Anda untuk bantuan lebih lanjut."

#. 60-70: token errors
#: ../common/rhnException.py:140
#, fuzzy
msgid ""
"\n"
"     The activation token specified could not be found on the server.\n"
"     Please retry with a valid key.\n"
"     "
msgstr ""
"Token aktivasi yang ditentukan tidak dapat ditemukan di server.\n"
"     Silakan coba lagi dengan kunci yang valid."

#: ../common/rhnException.py:144
#, fuzzy
msgid "Too many systems registered using this registration token"
msgstr "Terlalu banyak sistem yang terdaftar menggunakan token pendaftaran ini"

#: ../common/rhnException.py:145
#, fuzzy
msgid "Token contains invalid, obsoleted or insufficient settings"
msgstr "Token berisi pengaturan yang tidak valid, usang, atau tidak memadai"

#: ../common/rhnException.py:146
#, fuzzy
msgid "Conflicting activation tokens"
msgstr "Token aktivasi yang bertentangan"

#. 70-80: channel subscription errors
#: ../common/rhnException.py:149
#, fuzzy
msgid ""
"\n"
"     No matching base channel found for your system.\n"
"     "
msgstr "Tidak ada saluran dasar yang cocok yang ditemukan untuk sistem Anda."

#: ../common/rhnException.py:152
#, fuzzy
msgid ""
"\n"
"     You do not have subscription permission to the designated channel.\n"
"     Please refer to your organization's channel or organization\n"
"     administrators for further details.\n"
"     "
msgstr ""
"Anda tidak memiliki izin berlangganan ke saluran yang ditentukan.\n"
"     Lihat channel atau organisasi organisasi Anda\n"
"     administrator untuk rincian lebih lanjut."

#: ../common/rhnException.py:157
#, fuzzy
msgid "You can not unsubscribe from base channel."
msgstr "Anda tidak dapat berhenti berlangganan dari saluran dasar."

#: ../common/rhnException.py:158
#, fuzzy
msgid "SUSE Manager or Proxy channel can not be subscribed."
msgstr "Manajer SUSE atau saluran Proxy tidak dapat di subscribe."

#. 80-90: server group errors
#: ../common/rhnException.py:161
#, fuzzy
msgid "There was an error while trying to join the system to its groups"
msgstr ""
"Ada kesalahan saat mencoba untuk bergabung dengan sistem untuk kelompok-"
"kelompok"

#. 90-100: entitlement errors
#: ../common/rhnException.py:164
#, fuzzy
msgid "Unable to entitle system"
msgstr "Tidak dapat memberikan hak sistem"

#. 100-109: e-mail and uuid related faults
#: ../common/rhnException.py:167
#, fuzzy
msgid "Maximum e-mail length violation."
msgstr "Pelanggaran panjang e-mail maksimum."

#: ../common/rhnException.py:168
#, fuzzy
msgid "This system has been previously registered."
msgstr "Sistem ini sebelumnya telah terdaftar."

#: ../common/rhnException.py:169
#, fuzzy
msgid "Invalid username"
msgstr "Username tidak valid"

#. 140-159 applet errors
#: ../common/rhnException.py:172
#, fuzzy
msgid "Unable to look up server"
msgstr "Tidak dapat mencari server"

#. 160-179: OSAD errors
#: ../common/rhnException.py:175
#, fuzzy
msgid "Required argument is missing"
msgstr "Diperlukan argumen yang hilang."

#. 600-699: RHEL5+ EN errors
#: ../common/rhnException.py:178
#, fuzzy
msgid "No entitlement information tied to hardware"
msgstr "Tidak ada informasi penetapan yang terkait dengan perangkat keras"

#: ../common/rhnException.py:179
#, fuzzy
msgid "Installation number is not entitling"
msgstr "Nomor instalasi tidak entitling"

#. 700-799: Additional user input verification errors.
#: ../common/rhnException.py:182
#, fuzzy
msgid "Maximum username length violation"
msgstr "Pelanggaran panjang nama pengguna maksimum"

#: ../common/rhnException.py:183
#, fuzzy
msgid "Maximum password length violation"
msgstr "Pelanggaran panjang kata sandi maksimum"

#: ../common/rhnException.py:184
#, fuzzy
msgid "This user has read only API access. Action denied."
msgstr "Pengguna ini hanya memiliki akses API. Tindakan ditolak."

#: ../common/rhnException.py:186
#, fuzzy
msgid "System Name cannot be less than 1 character"
msgstr "Nama sistem tidak boleh kurang dari 1 karakter"

#. 1000-1999: Proxy specific errors:
#. issued by a Proxy to the client
#: ../common/rhnException.py:190
#, fuzzy
msgid "SUSE Manager Proxy error."
msgstr "Galat Proksi Manajer SUSE."

#: ../common/rhnException.py:191
#, fuzzy
msgid "SUSE Manager Proxy unable to login."
msgstr "SUSE Manager Proxy tidak dapat login."

#. issued by a SUSE Manager Server/Satellite to the proxy
#: ../common/rhnException.py:193
#, fuzzy
msgid ""
"\n"
"     SUSE Manager Proxy system ID does not match a Spacewalk Proxy Server\n"
"     in the database.\n"
"     "
msgstr ""
"ID sistem Proksi Manajer SUSE tidak cocok dengan Server Proksi Spacewalk\n"
"     dalam database."

#: ../common/rhnException.py:197
#, fuzzy
msgid "SUSE Manager Proxy session token is invalid."
msgstr "Token sesi Proksi Manajer SUSE tidak valid."

#: ../common/rhnException.py:198
#, fuzzy
msgid "SUSE Manager Proxy session token has expired."
msgstr "Token sesi Proxy Manajer SUSE telah kedaluwarsa."

#. 2000-2999: Red Hat Satellite specific errors:
#: ../common/rhnException.py:202
#, fuzzy
msgid ""
"\n"
"      user creation is not allowed via mgr_register;\n"
"     please contact your sysadmin to have your account created.\n"
"     "
msgstr ""
"pembuatan pengguna tidak diperbolehkan melalui mgr_register;\n"
"     silahkan hubungi sysadmin Anda untuk memiliki account Anda dibuat."

#: ../common/rhnException.py:206
#, fuzzy
msgid ""
"\n"
"     This SUSE Manager server is not allowed to use Inter Server Sync on "
"this server\n"
"     "
msgstr ""
"Server Manajer SUSE ini tidak diizinkan untuk menggunakan Inter Server Sync "
"di server ini"

#: ../common/rhnException.py:209
#, fuzzy
msgid ""
"\n"
"     Inter Server Sync is disabled on this SUSE Manager Server.\n"
"     "
msgstr "Inter Server sync dinonaktifkan pada Server manajer SUSE ini."

#. 3000-3999: XML dumper errors:
#: ../common/rhnException.py:214
#, fuzzy
msgid "Invalid datatype passed"
msgstr "Tipe data tidak valid dilewatkan"

#: ../common/rhnException.py:215
#, fuzzy
msgid "Unable to retrieve channel"
msgstr "Tidak dapat mengambil saluran"

#: ../common/rhnException.py:216
#, fuzzy
msgid "Unable to retrieve package"
msgstr "Tidak dapat mengambil paket"

#: ../common/rhnException.py:217
#, fuzzy
msgid "Unable to retrieve erratum"
msgstr "Tidak dapat mengambil erratum"

#: ../common/rhnException.py:218
#, fuzzy
msgid "File is missing"
msgstr "Berkas hilang"

#: ../common/rhnException.py:219
#, fuzzy
msgid "Function retrieval error"
msgstr "Galat pengambilan fungsi"

#: ../common/rhnException.py:220
#, fuzzy
msgid "Function execution error"
msgstr "Galat eksekusi fungsi"

#: ../common/rhnException.py:221
#, fuzzy
msgid "Missing version string"
msgstr "String versi hilang"

#: ../common/rhnException.py:222
#, fuzzy
msgid "Invalid version string"
msgstr "String versi tidak valid"

#: ../common/rhnException.py:223
#, fuzzy
msgid "Mismatching versions"
msgstr "Versi yang tidak cocok"

#: ../common/rhnException.py:224
#, fuzzy
msgid "Invalid channel version"
msgstr "Versi saluran tidak valid"

#: ../common/rhnException.py:225
#, fuzzy
msgid "No comps file for channel"
msgstr "Tidak ada file comps untuk saluran"

#: ../common/rhnException.py:226
#, fuzzy
msgid "Unable to retrieve comps file"
msgstr "Tidak dapat mengambil berkas comps"

#. 4000 - 4999: config management errors
#: ../common/rhnException.py:229
#, fuzzy
msgid "Configuration action missing"
msgstr "Tindakan konfigurasi hilang"

#: ../common/rhnException.py:230
#, fuzzy
msgid "File too large"
msgstr "Berkas terlalu besar"

#: ../common/rhnException.py:231
#, fuzzy
msgid "File contains binary data"
msgstr "File berisi data biner"

#: ../common/rhnException.py:232
#, fuzzy
msgid "Configuration channel is not empty"
msgstr "Saluran konfigurasi tidak kosong"

#: ../common/rhnException.py:233
#, fuzzy
msgid "Permission error"
msgstr "Galat izin"

#: ../common/rhnException.py:234
#, fuzzy
msgid "Content missing for configuration file"
msgstr "Konten hilang untuk file konfigurasi"

#: ../common/rhnException.py:235
#, fuzzy
msgid "Template delimiters not specified"
msgstr "Pembatas templat tidak ditentukan"

#: ../common/rhnException.py:236
#, fuzzy
msgid "Configuration channel does not exist"
msgstr "Saluran konfigurasi tidak ada"

#: ../common/rhnException.py:237
#, fuzzy
msgid "Configuration channel already exists"
msgstr "Saluran konfigurasi sudah ada"

#: ../common/rhnException.py:238
#, fuzzy
msgid "File missing from configuration channel"
msgstr "File hilang dari saluran konfigurasi"

#: ../common/rhnException.py:239
#, fuzzy
msgid "Different revision of this file is uploaded"
msgstr "Revisi yang berbeda dari file ini diunggah"

#: ../common/rhnException.py:240
#, fuzzy
msgid "File already uploaded to configuration channel"
msgstr "File yang sudah diunggah ke saluran konfigurasi"

#: ../common/rhnException.py:241
#, fuzzy
msgid "File size exceeds remaining quota space"
msgstr "Ukuran file melebihi ruang kuota yang tersisa"

#: ../common/rhnException.py:242
#, fuzzy
msgid "Full path of file must be specified"
msgstr "Path lengkap file harus ditentukan"

#: ../common/rhnException.py:243
#, fuzzy
msgid "Invalid revision number"
msgstr "Nomor revisi tidak valid"

#: ../common/rhnException.py:244
#, fuzzy
msgid "Cannot compare files of different file type"
msgstr "Tidak dapat membandingkan berkas jenis berkas yang berbeda"

#. 5000 - 5999: client content uploading errors
#. 5000 - 5099: crash reporting errors
#: ../common/rhnException.py:248
#, fuzzy
msgid "Crash information is invalid or incomplete"
msgstr "Informasi kerusakan tidak valid atau tidak lengkap"

#: ../common/rhnException.py:249
#, fuzzy
msgid "Crash file information is invalid or incomplete"
msgstr "Informasi berkas lumpuh tidak valid atau tidak lengkap"

#: ../common/rhnException.py:250
#, fuzzy
msgid "Error composing crash directory path"
msgstr "Galat saat menyusun jalur direktori crash"

#: ../common/rhnException.py:251
#, fuzzy
msgid "Error composing crash file path"
msgstr "Galat saat menyusun jalur berkas lumpuh"

#: ../common/rhnException.py:252 ../common/rhnException.py:259
#, fuzzy
msgid "Invalid content encoding"
msgstr "Pengkodean konten tidak valid"

#: ../common/rhnException.py:253
#, fuzzy
msgid "Invalid crash name"
msgstr "Nama crash tidak valid"

#: ../common/rhnException.py:254
#, fuzzy
msgid "Crash reporting is disabled for this organization"
msgstr "Pelaporan kerusakan dinonaktifkan untuk organisasi ini"

#. 5100 - 5199: scap results reporting error
#: ../common/rhnException.py:256
#, fuzzy
msgid "SCAP results file transfer is invalid or incomplete"
msgstr "Transfer berkas hasil SCAP tidak valid atau tidak lengkap"

#: ../common/rhnException.py:257
#, fuzzy
msgid "Error composing directory path for detailed SCAP results"
msgstr "Galat saat menyusun jalur direktori untuk hasil SCAP terperinci"

#: ../common/rhnException.py:258
#, fuzzy
msgid "Error composing file path for detailed SCAP results"
msgstr "Galat saat menyusun jalur berkas untuk hasil SCAP rinci"

#: ../common/rhnException.py:306
#, fuzzy
msgid ""
"\n"
"     An error has occurred while processing your request. If this problem\n"
"     persists please enter a bug report at scc.suse.com.\n"
"     If you choose to submit the bug report, please be sure to include\n"
"     details of what you were trying to do when this error occurred and\n"
"     details on how to reproduce this problem.\n"
msgstr ""
"Telah terjadi galat saat memproses permintaan Anda. Jika masalah ini\n"
"     tetap ada, silakan masukkan laporan bug di scc.suse.com.\n"
"     Jika Anda memilih untuk mengirimkan laporan bug, pastikan untuk\n"
"     rincian dari apa yang Anda coba lakukan ketika kesalahan ini terjadi "
"dan\n"
"     rincian tentang cara mereproduksi masalah ini."

#: ../common/rhnException.py:358
#, fuzzy, python-format
msgid ""
"Error Message:\n"
"    %s\n"
msgstr "Pesan kesalahan tak tertangani: %s"

#: ../common/rhnException.py:360
#, fuzzy, python-format
msgid "Error Class Code: %s\n"
msgstr "Kode Kelas Galat: %s"

#: ../common/rhnException.py:363
#, fuzzy, python-format
msgid "Error Class Info: %s\n"
msgstr "Galat Info Kelas: %s"

#: ../common/rhnException.py:365
#, fuzzy, python-format
msgid "Explanation: %s"
msgstr "Penjelasan: %s"

#: ../common/rhnTB.py:205
#, fuzzy, python-format
msgid "ERROR: %s %s: %s"
msgstr "GALAT: %s %s: %s"

#: ../common/rhnTB.py:207
#, fuzzy, python-format
msgid "TRACEBACK: %s"
msgstr "TRACEBACK: %s (Dalam Bahasa Inggris)"

#: ../satellite_tools/disk_dumper/iss.py:1294
#: ../satellite_tools/disk_dumper/iss.py:1298
#, fuzzy, python-format
msgid "format of %s should be at least YYYYMMDD.\n"
msgstr "format dari %s harus setidaknya YYYYMMDD."

#: ../satellite_tools/messages.py:26
#, fuzzy, python-format
msgid ""
"\n"
"ERROR: executing step %s. Error is:\n"
"%s\n"
msgstr ""
"GALAT: menjalankan langkah %s. Galat adalah:\n"
"%s"

#: ../satellite_tools/messages.py:31
#, fuzzy, python-format
msgid ""
"\n"
"ERROR: there was a problem accessing the channel data from your temporary\n"
"       repository. Did you migrate all of the data from the channel ISOs\n"
"       do this directory? If so, please recheck the channels ISOs, ensure\n"
"       that you have them all, and then iteratively remount and repopulate\n"
"       the temporary repository (%s).\n"
msgstr ""
"ERROR: ada masalah saat mengakses data channel dari\n"
"       Repositori. Apakah Anda memigrasikan semua data dari ISO saluran\n"
"       melakukan direktori ini? Jika demikian, silakan periksa kembali "
"saluran ISOs, pastikan\n"
"       bahwa Anda memiliki mereka semua, dan kemudian berulang-ulang dan "
"mengisi kembali\n"
"       repositori sementara (%s)."

#: ../satellite_tools/messages.py:39
#, fuzzy, python-format
msgid ""
"\n"
"ERROR: there was a problem synchronizing the information.\n"
"       Error message: %s\n"
msgstr ""
"GALAT: ada masalah sinkronisasi informasi.\n"
"       Pesan galat: %s"

#: ../satellite_tools/messages.py:44
#, fuzzy, python-format
msgid ""
"\n"
"ERROR: There was a problem communicating with the ISS Master.\n"
"       Depending on the specific error details, please review your "
"configuration,\n"
"       basic network connectivity, and/or name resolution, and try again.\n"
"       Error message: %s\n"
msgstr ""
"GALAT: Ada masalah berkomunikasi dengan ISS Master.\n"
"       Tergantung pada rincian kesalahan tertentu, silakan tinjau "
"konfigurasi Anda,\n"
"       konektivitas jaringan dasar, dan/atau resolusi nama, dan coba lagi.\n"
"       Pesan galat: %s"

#: ../satellite_tools/messages.py:51
#, fuzzy, python-format
msgid ""
"\n"
"ERROR: There was a problem communicating with the ISS Master.\n"
"       If the master satellite is older than v5.3, it does not have ISS "
"capability.\n"
"       Otherwise, depending on the specific error details, please review "
"your\n"
"       configuration, basic network connectivity, and/or name resolution, "
"and try again.\n"
"       Error message: %s\n"
msgstr ""
"GALAT: Ada masalah berkomunikasi dengan ISS Master.\n"
"       Jika satelit master lebih tua dari v5.3, itu tidak memiliki kemampuan "
"ISS.\n"
"       Jika tidak, tergantung pada rincian kesalahan tertentu, harap tinjau\n"
"       konfigurasi, konektivitas jaringan dasar, dan/atau resolusi nama, dan "
"coba lagi.\n"
"       Pesan galat: %s"

#: ../satellite_tools/messages.py:59
#, fuzzy, python-format
msgid ""
"\n"
"ERROR: a child-channel cannot be synced without its parent being synced as\n"
"       well. A parent needs to be either (a) previously synced or (b) "
"synced\n"
"       in tandem with the desired child-channel. Missing parents for this\n"
"       transaction:\n"
"       %s\n"
msgstr ""
"GALAT: saluran anak tidak dapat disinkronkan tanpa induknya disinkronkan "
"sebagai\n"
"       Baik. Orang tua harus (a) sebelumnya disinkronkan atau (b) "
"disinkronkan\n"
"       bersama-sama dengan saluran anak yang diinginkan. Orang tua yang "
"hilang untuk ini\n"
"       Transaksi:\n"
"       %s"

#: ../satellite_tools/messages.py:67
#, fuzzy, python-format
msgid ""
"\n"
"ERROR: you are not entitled to sync a channel in this set of channels.\n"
"Please contact your SUSE Manager contact\n"
"%s"
msgstr ""
"ERROR: Anda tidak berhak menyinkronkan saluran dalam serangkaian saluran "
"ini.\n"
"Silakan hubungi kontak SUSE Manager Anda\n"
"%s"

#: ../satellite_tools/messages.py:72
#, fuzzy, python-format
msgid "  ERROR: not enough free space (%s KB) on device."
msgstr "GALAT: tidak cukup ruang kosong (%s KB) pada perangkat."

#: ../satellite_tools/messages.py:74
#, fuzzy, python-format
msgid "    %3d/%s Fetch successful: %s (%s bytes)"
msgstr "%3d/%s Fetch berhasil: %s (%s bytes)"

#: ../satellite_tools/messages.py:75
#, fuzzy, python-format
msgid "    Extinct package:  %s"
msgstr "Paket punah: %s"

#: ../satellite_tools/messages.py:76
#, fuzzy, python-format
msgid "   Total size: %s"
msgstr "Total ukuran: %s"

#: ../satellite_tools/messages.py:77
#, fuzzy, python-format
msgid "      Downloaded %s of %s. Estimated remaining time: %s"
msgstr "Diunduh %s dari %s. Perkiraan sisa waktu: %s"

#: ../satellite_tools/messages.py:78
#, fuzzy, python-format
msgid "    Fetch unsuccessful: %s"
msgstr "Fetch gagal: %s"

#: ../satellite_tools/messages.py:79
#, fuzzy, python-format
msgid "   RPM fetch summary: %s"
msgstr "Ringkasan pengambilan RPM: %s"

#: ../satellite_tools/messages.py:80
#, fuzzy, python-format
msgid "       success: %d"
msgstr "sukses: %d"

#: ../satellite_tools/messages.py:81
#, fuzzy, python-format
msgid "       failed:  %d"
msgstr "Impor yang gagal (%d)"

#: ../satellite_tools/messages.py:82
#, fuzzy, python-format
msgid "       extinct: %d"
msgstr "punah: %d"

#: ../satellite_tools/messages.py:84
#, fuzzy, python-format
msgid "   Retrieving / parsing *relevant* package metadata: %s (%s)"
msgstr "Mengambil / mem-parsing *relevan* metadata paket: %s (%s)"

#: ../satellite_tools/messages.py:85
#, fuzzy, python-format
msgid "   Retrieving / parsing errata data: %s (%s)"
msgstr "Mengambil / mengurai data errata: %s (%s)"

#: ../satellite_tools/messages.py:86
#, fuzzy, python-format
msgid "   Retrieving / parsing kickstart data: %s (%s)"
msgstr "Mengambil / mengurai data kickstart: %s (%s)"

#: ../satellite_tools/messages.py:87
#, fuzzy, python-format
msgid "   Retrieving / parsing kickstart tree files: %s (%s)"
msgstr "Mengambil / mengurai berkas pohon kickstart: %s (%s)"

#: ../satellite_tools/messages.py:88
#, fuzzy, python-format
msgid "   Importing *relevant* package metadata: %s (%s)"
msgstr "Mengimpor metadata paket *relevant*: %s (%s)"

#: ../satellite_tools/messages.py:89
#, fuzzy
msgid "   * WARNING: this may be a slow process."
msgstr "* PERINGATAN: ini mungkin proses yang lambat."

#: ../satellite_tools/messages.py:90
#, fuzzy
msgid "Linking packages to channels"
msgstr "Menautkan paket ke saluran"

#: ../satellite_tools/messages.py:91
#, fuzzy, python-format
msgid "   Importing *relevant* errata: %s (%s)"
msgstr "Mengimpor *relevan* errata: %s (%s)"

#: ../satellite_tools/messages.py:92
#, fuzzy
msgid "   No new kickstartable tree to import"
msgstr "Tidak ada pohon kickstartable baru untuk mengimpor"

#: ../satellite_tools/messages.py:93
#, fuzzy, python-format
msgid "Importing kickstartable trees (%d)"
msgstr "Mengimpor pohon kickstartable (%d)"

#: ../satellite_tools/messages.py:94
#, fuzzy, python-format
msgid "Imported kickstartable trees (%d)"
msgstr "Pohon kickstartable impor (%d)"

#: ../satellite_tools/rhn_satellite_activate.py:198
#, fuzzy, python-format
msgid ""
"\"%s\" (specified in commandline)\n"
"could not be opened and read:\n"
"%s"
msgstr ""
"\"%s\" (ditentukan dalam baris perintah)\n"
"tidak dapat dibuka dan dibaca:\n"
"%s"

#: ../satellite_tools/rhn_satellite_activate.py:205
#, fuzzy, python-format
msgid ""
"\"%s\" could not be opened\n"
"and/or written to:\n"
"%s"
msgstr ""
"\"%s\" tidak dapat dibuka\n"
"dan/atau tertulis kepada:\n"
"%s"

#: ../satellite_tools/reposync.py:111 ../satellite_tools/satsync.py:439
#, fuzzy
msgid "+++ sending log as an email +++"
msgstr "+++ mengirim log sebagai email +++"

#: ../satellite_tools/reposync.py:114
#, fuzzy, python-format
msgid "%s sync. report from %s"
msgstr "Sinkronisasi %s. laporan dari %s"

#: ../satellite_tools/reposync.py:124 ../satellite_tools/satsync.py:449
#, fuzzy
msgid "+++ email requested, but there is nothing to send +++"
msgstr "+++ email yang diminta, tetapi tidak ada yang bisa dikirim +++"

#: ../satellite_tools/satsync.py:258
#, fuzzy, python-format
msgid "No handler for step %s"
msgstr "Tidak ada handler untuk langkah %s"

#. Try one more time - this time it should be faster since
#. everything should be cached
#: ../satellite_tools/satsync.py:270
#, fuzzy
msgid "Environment changed, trying again..."
msgstr "Lingkungan berubah, mencoba lagi ..."

#: ../satellite_tools/satsync.py:277
#, fuzzy
msgid "ERROR: Encountered IntegrityError: \n"
msgstr "GALAT: Mengalami IntegrityError:"

#: ../satellite_tools/satsync.py:285
#, fuzzy
msgid "Repeated failures"
msgstr "Kegagalan berulang"

#: ../satellite_tools/satsync.py:290
#, fuzzy, python-format
msgid ""
"    Import complete:\n"
"        Begin time: %s\n"
"        End time:   %s\n"
"        Elapsed:    %s\n"
"          "
msgstr ""
"Impor selesai:\n"
"        Waktu mulai: %s\n"
"        Waktu akhir: %s\n"
"        Berlalu: %s"

#: ../satellite_tools/satsync.py:311
#, fuzzy
msgid "hours"
msgstr "jam"

#: ../satellite_tools/satsync.py:311
#, fuzzy
msgid "minutes"
msgstr "menit"

#: ../satellite_tools/satsync.py:311
#, fuzzy
msgid "seconds"
msgstr "detik"

#: ../satellite_tools/satsync.py:357
#, fuzzy
msgid ""
"ERROR: The dump is missing package data, use --no-rpms to skip this step or "
"fix the content to include package data."
msgstr ""
"GALAT: Dump hilang paket data, gunakan --no-rpms untuk melewati langkah ini "
"atau memperbaiki konten untuk menyertakan paket data."

#. the orgs() method doesn't exist; that's fine we just
#. won't sync the orgs
#: ../satellite_tools/satsync.py:404
#, fuzzy
msgid "The SUSE Manager master does not support syncing orgs data."
msgstr "Master Manajer SUSE tidak mendukung sinkronisasi data organisasi."

#: ../satellite_tools/satsync.py:404
#, fuzzy
msgid "Skipping..."
msgstr "Melewatkan: %s\n"

#: ../satellite_tools/satsync.py:442
#, fuzzy, python-format
msgid "SUSE Manager Inter Server sync. report from %s"
msgstr "Sinkronisasi SUSE Manager Inter Server. laporan dari %s"

#: ../satellite_tools/satsync.py:521
#, fuzzy
msgid " - file-system synchronization"
msgstr "- Sinkronisasi sistem file"

#: ../satellite_tools/satsync.py:533 ../satellite_tools/satsync.py:541
#, fuzzy
msgid " - live synchronization"
msgstr "- Sinkronisasi langsung"

#: ../satellite_tools/satsync.py:534
#, fuzzy, python-brace-format
msgid ""
"ERROR: Live content synchronizing with RHN Classic Hosted is no longer "
"supported.\n"
"Please use the cdn-sync command instead unless you are attempting to sync "
"from another {PRODUCT_NAME} via Inter-Server-Sync (ISS), or from local "
"content on disk via Channel Dump ISOs."
msgstr ""
"KESALAHAN: Sinkronisasi konten langsung dengan RHN Classic Hosted tidak lagi "
"didukung.\n"
"Silakan gunakan perintah cdn-sync sebagai gantinya kecuali Jika Anda mencoba "
"untuk menyinkronkan dari Satelit lain melalui Inter-Satelite-Sync (ISS), "
"atau dari konten lokal pada disk melalui Channel Dump ISOs."

#: ../satellite_tools/satsync.py:542
#, fuzzy, python-format
msgid "   url: %s"
msgstr "URL: %s"

#: ../satellite_tools/satsync.py:543
#, fuzzy, python-format
msgid "   debug/output level: %s"
msgstr "tingkat debug/keluaran: %s"

#: ../satellite_tools/satsync.py:553
#, fuzzy
msgid "ERROR: this server must be registered with SUSE Manager."
msgstr "GALAT: server ini harus terdaftar di SUSE Manager."

#: ../satellite_tools/satsync.py:575
#, fuzzy, python-format
msgid "Retrieving / parsing %s data"
msgstr "Mengambil / mengurai data %s"

#: ../satellite_tools/satsync.py:585 ../satellite_tools/satsync.py:600
#, fuzzy
msgid "*** SYSTEM INTERRUPT CALLED ***"
msgstr "SISTEM INTERUPSI DISEBUT ***"

#: ../satellite_tools/satsync.py:593
#, fuzzy
msgid "ERROR: fatal parser exception occurred "
msgstr "GALAT: pengecualian parser fatal terjadi"

#: ../satellite_tools/satsync.py:594
#, fuzzy, python-format
msgid "(line: %s, col: %s msg: %s)"
msgstr "(baris: %s, col: %s msg: %s)"

#: ../satellite_tools/satsync.py:598
#, fuzzy, python-format
msgid "ERROR: parser exception occurred: %s"
msgstr "GALAT: pengecualian parser terjadi: %s"

#: ../satellite_tools/satsync.py:603
#, fuzzy
msgid "ERROR: exception (during parse) occurred: "
msgstr "GALAT: pengecualian (selama parse) terjadi:"

#: ../satellite_tools/satsync.py:604
#, fuzzy, python-format
msgid ""
"   Encountered some errors with %s data (see logs (%s) for more information)"
msgstr ""
"Mengalami beberapa kesalahan dengan data %s (lihat log (%s) untuk informasi "
"lebih lanjut)"

#: ../satellite_tools/satsync.py:606
#, fuzzy, python-format
msgid "   Encountered some errors with %s data:"
msgstr "Mengalami beberapa kesalahan dengan data %s:"

#: ../satellite_tools/satsync.py:607
#, fuzzy, python-format
msgid "   ------- %s PARSE/IMPORT ERROR -------"
msgstr "------- GALAT PARSE/IMPORT %s -------"

#: ../satellite_tools/satsync.py:609
#, fuzzy
msgid "   ---------------------------------------"
msgstr "---------------------------------------"

#: ../satellite_tools/satsync.py:612
#, fuzzy, python-format
msgid "%s data complete"
msgstr "%s data selesai"

#: ../satellite_tools/satsync.py:665
#, fuzzy
msgid "Retrieving / parsing channel data"
msgstr "Mengambil / mengurai data saluran"

#: ../satellite_tools/satsync.py:718
#, fuzzy
msgid "Channel data complete"
msgstr "Data saluran selesai"

#: ../satellite_tools/satsync.py:736
#, fuzzy, python-format
msgid "%10s import from %s"
msgstr "%10s mengimpor dari %s"

#: ../satellite_tools/satsync.py:739
#, fuzzy, python-format
msgid "%10s import from %s - %s"
msgstr "%10s mengimpor dari %s - %s"

#: ../satellite_tools/satsync.py:743
#, fuzzy, python-format
msgid "%10s"
msgstr "-10s"

#: ../satellite_tools/satsync.py:750
#, fuzzy
msgid "p"
msgstr "P"

#: ../satellite_tools/satsync.py:752
#, fuzzy
msgid "."
msgstr "."

#: ../satellite_tools/satsync.py:758
#, fuzzy
msgid "   p = previously imported/synced channel"
msgstr "p = saluran yang diimpor/disinkronkan sebelumnya"

#: ../satellite_tools/satsync.py:759
#, fuzzy
msgid "   . = channel not yet imported/synced"
msgstr ". = saluran belum diimpor / disinkronkan"

#: ../satellite_tools/satsync.py:765
#, fuzzy
msgid "   e = channel no longer supported (end-of-service)"
msgstr "e = saluran tidak lagi didukung (akhir layanan)"

#: ../satellite_tools/satsync.py:767
#, fuzzy
msgid "   ? = channel label invalid --- typo?"
msgstr "? = label saluran tidak --- kesalahan ketik?"

#: ../satellite_tools/satsync.py:771
#, fuzzy, python-format
msgid "   %s:"
msgstr "Untuk: %s"

#: ../satellite_tools/satsync.py:772
#, fuzzy, python-format
msgid "      %s %-40s %4s %s"
msgstr "%s %-40s %4s %s"

#: ../satellite_tools/satsync.py:773
#, fuzzy
msgid "base-channels"
msgstr "saluran dasar"

#: ../satellite_tools/satsync.py:786 ../satellite_tools/satsync.py:1603
#, fuzzy
msgid "NONE RELEVANT"
msgstr "TIDAK ADA YANG RELEVAN"

#: ../satellite_tools/satsync.py:808
#, fuzzy
msgid "end-of-service"
msgstr "akhir layanan"

#: ../satellite_tools/satsync.py:809
#, fuzzy
msgid "e"
msgstr "e"

#: ../satellite_tools/satsync.py:815
#, fuzzy
msgid "   typos:"
msgstr "Typos:"

#: ../satellite_tools/satsync.py:816
#, fuzzy
msgid "?"
msgstr "?"

#: ../satellite_tools/satsync.py:831
#, fuzzy, python-format
msgid "XXX: imported channels: %s"
msgstr "XXX: saluran yang diimpor: %s"

#: ../satellite_tools/satsync.py:832
#, fuzzy, python-format
msgid "XXX:   cached channels: %s"
msgstr "XXX: saluran cache: %s"

#: ../satellite_tools/satsync.py:837
#, fuzzy
msgid "XXX: list channels called"
msgstr "XXX: daftar saluran yang disebut"

#: ../satellite_tools/satsync.py:855
#, fuzzy
msgid "ERROR: these channels either do not exist or are not available:"
msgstr "GALAT: saluran ini tidak ada atau tidak tersedia:"

#: ../satellite_tools/satsync.py:859
#, fuzzy, python-format
msgid "       (to see a list of channel labels: %s --list-channels)"
msgstr "(untuk melihat daftar label saluran: %s --list-channels)"

#: ../satellite_tools/satsync.py:904
#, fuzzy, python-format
msgid "   Retrieving / parsing short package metadata: %s (%s)"
msgstr "Mengambil / mengurai metadata paket pendek: %s (%s)"

#: ../satellite_tools/satsync.py:965
#, fuzzy, python-format
msgid "Diffing package metadata (what's missing locally?): %s"
msgstr "Metadata paket diffing (apa yang hilang secara lokal?): %s"

#: ../satellite_tools/satsync.py:971 ../satellite_tools/satsync.py:1254
#, fuzzy
msgid "Diffing:    "
msgstr "Diffing:"

#: ../satellite_tools/satsync.py:988
#, fuzzy
msgid "ERROR: incremental dump skipped"
msgstr "GALAT: inkremental dump dilewati"

#: ../satellite_tools/satsync.py:1079
#, fuzzy
msgid "Downloading rpm packages"
msgstr "Mengunduh paket rpm"

#: ../satellite_tools/satsync.py:1084
#, fuzzy, python-format
msgid "   Fetching any missing RPMs: %s (%s)"
msgstr "Mengambil RPM yang hilang: %s (%s)"

#: ../satellite_tools/satsync.py:1085 ../satellite_tools/satsync.py:1153
#, fuzzy
msgid "NONE MISSING"
msgstr "TIDAK ADA YANG HILANG"

#: ../satellite_tools/satsync.py:1094
#, fuzzy
msgid "Processing rpm packages complete"
msgstr "Memproses paket rpm lengkap"

#: ../satellite_tools/satsync.py:1121
#, fuzzy
msgid "Downloading package metadata"
msgstr "Mengunduh metadata paket"

#: ../satellite_tools/satsync.py:1146
#, fuzzy
msgid "Downloading srpm packages"
msgstr "Mengunduh paket srpm"

#: ../satellite_tools/satsync.py:1152
#, fuzzy, python-format
msgid "   Fetching any missing SRPMs: %s (%s)"
msgstr "Mengambil SRPM yang hilang: %s (%s)"

#: ../satellite_tools/satsync.py:1249
#, fuzzy, python-format
msgid "Diffing source package metadata (what's missing locally?): %s"
msgstr "Metadata paket sumber diffing (apa yang hilang secara lokal?): %s"

#: ../satellite_tools/satsync.py:1260
#, fuzzy
msgid "Downloading source package metadata"
msgstr "Mengunduh metadata paket sumber"

#: ../satellite_tools/satsync.py:1338 ../satellite_tools/satsync.py:2027
#, fuzzy, python-format
msgid "Unable to save file %s: %s"
msgstr "Tidak dapat menyimpan berkas %s: %s"

#. for
#. Retried a number of times and it still failed; log the
#. file as being failed and move on
#: ../satellite_tools/satsync.py:1345
#, fuzzy, python-format
msgid "Failed to fetch file %s"
msgstr "Gagal mengambil berkas %s"

#: ../satellite_tools/satsync.py:1350
#, fuzzy
msgid "Downloading kickstartable trees metadata"
msgstr "Mengunduh metadata pohon kickstartable"

#: ../satellite_tools/satsync.py:1365
#, fuzzy
msgid "Downloading kickstartable trees files"
msgstr "Mengunduh file pohon kickstartable"

#: ../satellite_tools/satsync.py:1504
#, fuzzy, python-format
msgid "Grabbing all patches for channel %s"
msgstr "Meraih semua patch untuk saluran %s"

#: ../satellite_tools/satsync.py:1554
#, fuzzy
msgid "Downloading patch data"
msgstr "Mengunduh data patch"

#: ../satellite_tools/satsync.py:1556
#, fuzzy
msgid "Forcing download of all patch data for requested channels."
msgstr "Memaksa pengunduhan semua data patch untuk saluran yang diminta."

#: ../satellite_tools/satsync.py:1571
#, fuzzy
msgid "Downloading patch data complete"
msgstr "Mengunduh data patch selesai"

#: ../satellite_tools/satsync.py:1576 ../satellite_tools/satsync.py:1597
#, fuzzy
msgid "Downloading:"
msgstr "Unduh:"

#: ../satellite_tools/satsync.py:1579
#, fuzzy
msgid " - complete"
msgstr "%s Selesai"

#: ../satellite_tools/satsync.py:1624
#, fuzzy
msgid "Importing source package metadata"
msgstr "Mengimpor metadata paket sumber"

#: ../satellite_tools/satsync.py:1627
#, fuzzy
msgid "Importing package metadata"
msgstr "Mengimpor metadata paket"

#: ../satellite_tools/satsync.py:1635
#, fuzzy
msgid "Importing:  "
msgstr "Mengimpor..."

#. not in the cache
#: ../satellite_tools/satsync.py:1703
#, fuzzy
msgid ""
"Package Not Found in Cache, Clear the Cache "
"to                                  Regenerate it."
msgstr ""
"Paket tidak ditemukan di Cache, Kosongkan Cache untuk meregenerasi itu.Paket "
"tidak ditemukan di Cache, Kosongkan Cache untuk meregenerasi itu."

#: ../satellite_tools/satsync.py:1709
#, fuzzy
msgid "Importing channel patches"
msgstr "Mengimpor patch saluran"

#: ../satellite_tools/satsync.py:1714
#, fuzzy, python-format
msgid "Importing %s patches for channel %s."
msgstr "Mengimpor patch %s untuk saluran %s."

#. Incomplete configuration
#: ../satellite_tools/satsync.py:2113
#, fuzzy
msgid "ERROR: server.mount_point not set in the configuration file"
msgstr "GALAT: server.mount_point tidak ditetapkan dalam berkas konfigurasi"

#: ../satellite_tools/satsync.py:2117
#, fuzzy, python-format
msgid "ERROR: server.mount_point %s do not exist"
msgstr "GALAT: server.mount_point %s tidak ada"

#: ../satellite_tools/satsync.py:2122
#, fuzzy, python-format
msgid "ERROR: path under server.mount_point (%s)  do not exist"
msgstr "GALAT: jalur di server.mount_point (%s) tidak ada"

#: ../satellite_tools/satsync.py:2199
#, fuzzy, python-format
msgid "Commandline: %s"
msgstr "Baris perintah: %s"

#: ../satellite_tools/satsync.py:2202
#, fuzzy, python-format
msgid ""
"DEBUG ONLY: max. batch-size for XML/database-import processing (1..%s).\"man "
"satellite-sync\" for more information."
msgstr ""
"DEBUG HANYA: max. batch-size untuk XML/database-import processing (1..%s).\" "
"manusia satelit-sync\" untuk informasi lebih lanjut."

#: ../satellite_tools/satsync.py:2205
#, fuzzy
msgid "alternative SSL CA Cert (fullpath to cert file)"
msgstr "alternatif SSL CA Cert (fullpath ke cert file)"

#: ../satellite_tools/satsync.py:2207
#, fuzzy
msgid "process data for this channel only"
msgstr "data proses untuk saluran ini saja"

#: ../satellite_tools/satsync.py:2209
#, fuzzy
msgid ""
"disk dump will be considered to be a full export; see \"man satellite-sync\" "
"for more information."
msgstr ""
"dump disk akan dianggap sebagai ekspor penuh; lihat \"man satellite-sync\" "
"untuk informasi lebih lanjut."

#: ../satellite_tools/satsync.py:2212
#, fuzzy
msgid "existing custom channels will also be synced (unless -c is used)"
msgstr "saluran kustom yang ada juga akan disinkronkan (kecuali -c digunakan)"

#: ../satellite_tools/satsync.py:2214
#, fuzzy, python-format
msgid ""
"override debug level set in /etc/rhn/rhn.conf (which is currently set at %s)."
msgstr ""
"menimpa tingkat debug yang ditetapkan dalam /etc/rhn/rhn.conf (yang saat ini "
"disedingkan di %s)."

#: ../satellite_tools/satsync.py:2216
#, fuzzy, python-format
msgid "requested version of XML dump (default: %s)"
msgstr "versi yang diminta dari dump XML (default: %s)"

#: ../satellite_tools/satsync.py:2218
#, fuzzy
msgid "e-mail a report of what was synced/imported"
msgstr "e-mail laporan tentang apa yang disinkronkan / diimpor"

#: ../satellite_tools/satsync.py:2220
#, fuzzy
msgid "forcibly process all (not a diff of) patch metadata"
msgstr "secara paksa memproses semua (bukan diff) patch metadata"

#: ../satellite_tools/satsync.py:2222
#, fuzzy
msgid "Do not use an http proxy under any circumstances."
msgstr "Jangan gunakan proxy http dalam keadaan apa pun."

#: ../satellite_tools/satsync.py:2224
#, fuzzy
msgid "alternative http proxy (hostname:port)"
msgstr "alternatif http proxy (hostname:port)"

#: ../satellite_tools/satsync.py:2226
#, fuzzy
msgid "alternative http proxy username"
msgstr "alternatif http proxy username"

#: ../satellite_tools/satsync.py:2228
#, fuzzy
msgid "alternative http proxy password"
msgstr "kata sandi proxy http alternatif"

#: ../satellite_tools/satsync.py:2230
#, fuzzy
msgid "parent SUSE Manager to import content from"
msgstr "induk SUSE Manager untuk mengimpor konten dari"

#: ../satellite_tools/satsync.py:2232
#, fuzzy
msgid "list all available channels and exit"
msgstr "daftar semua saluran yang tersedia dan keluar"

#: ../satellite_tools/satsync.py:2234
#, fuzzy
msgid "help on all error codes mgr-inter-sync returns"
msgstr "bantuan pada semua kode kesalahan mgr-inter-sync kembali"

#: ../satellite_tools/satsync.py:2236
#, fuzzy
msgid "source mount point for import - disk update only"
msgstr "titik pasang sumber untuk impor - hanya pembaruan disk"

#: ../satellite_tools/satsync.py:2238
#, fuzzy
msgid "do not process patch data"
msgstr "jangan memproses data patch"

#: ../satellite_tools/satsync.py:2240
#, fuzzy
msgid "do not process kickstart data (provisioning only)"
msgstr "jangan memproses data kickstart (hanya penyediaan)"

#: ../satellite_tools/satsync.py:2242
#, fuzzy
msgid "do not process full package metadata"
msgstr "tidak memproses metadata paket lengkap"

#: ../satellite_tools/satsync.py:2244
#, fuzzy
msgid "do not download, or process any RPMs"
msgstr "tidak mengunduh, atau memproses RPM apa pun"

#: ../satellite_tools/satsync.py:2246
#, fuzzy
msgid "turn off SSL (not recommended)"
msgstr "menonaktifkan SSL (tidak disarankan)"

#: ../satellite_tools/satsync.py:2248
#, fuzzy
msgid "org to which the sync imports data. defaults to the admin account"
msgstr "organisasi tempat sinkronisasi mengimpor data. default ke akun admin"

#: ../satellite_tools/satsync.py:2250
#, fuzzy
msgid "print the configuration and exit"
msgstr "mencetak konfigurasi dan keluar"

#: ../satellite_tools/satsync.py:2252
#, fuzzy
msgid "alternative server with which to connect (hostname)"
msgstr "server alternatif yang digunakan untuk menghubungkan (nama host)"

#: ../satellite_tools/satsync.py:2254
#, fuzzy
msgid "synchronize to this step (man satellite-sync for more info)"
msgstr ""
"sinkronisasi ke langkah ini (man satellite-sync untuk info lebih lanjut)"

#: ../satellite_tools/satsync.py:2256
#, fuzzy
msgid "write complete data to tempfile before streaming to remainder of app"
msgstr "menulis data lengkap ke tempfile sebelum streaming ke sisa aplikasi"

#: ../satellite_tools/satsync.py:2258
#, fuzzy
msgid "DEBUG ONLY: alternative path to digital system id"
msgstr "DEBUG HANYA: jalur alternatif ke id sistem digital"

#: ../satellite_tools/satsync.py:2260
#, fuzzy
msgid "alternative email address(es) for sync output (--email option)"
msgstr "alamat email alternatif untuk output sinkronisasi (--opsi email)"

#: ../satellite_tools/satsync.py:2262
#, fuzzy
msgid "do not remove rpms when importing from local dump"
msgstr "tidak menghapus rpm saat mengimpor dari dump lokal"

#: ../satellite_tools/satsync.py:2264
#, fuzzy
msgid ""
"the fully qualified domain name of the master Satellite. Valid with --mount-"
"point only. Required if you want to import org data and channel permissions."
msgstr ""
"nama domain yang memenuhi syarat dari satelit master. Hanya valid dengan --"
"mount-point. Diperlukan jika Anda ingin mengimpor data organisasi dan izin "
"saluran."

#: ../satellite_tools/satsync.py:2274
#, fuzzy, python-format
msgid "ERROR: these arguments make no sense in this context (try --help): %s"
msgstr ""
"GALAT: argumen ini tidak masuk akal dalam konteks ini (coba --help): "
"%sGALAT: argumen ini tidak masuk akal dalam konteks ini (coba --help): %s"

#. An SQL error is fatal... crash and burn
#: ../satellite_tools/satsync.py:2286
#, fuzzy, python-format
msgid "ERROR: Can't connect to the database: %s"
msgstr "GALAT: Tidak dapat menyambung ke pangkalan data: %s"

#: ../satellite_tools/satsync.py:2287
#, fuzzy
msgid "ERROR: Check if your database is running."
msgstr "GALAT: Periksa apakah database Anda berjalan."

#: ../satellite_tools/satsync.py:2322
#, fuzzy, python-format
msgid "ERROR: --debug-level takes an in integer value within the range %s."
msgstr "GALAT: --debug-level mengambil nilai bilangan bulat dalam kisaran %s."

#: ../satellite_tools/satsync.py:2324
#, fuzzy
msgid "  0  - little logging/messaging."
msgstr "0 - sedikit penebangan / pesan."

#: ../satellite_tools/satsync.py:2325
#, fuzzy
msgid "  1  - minimal logging/messaging."
msgstr "1 - minimal penebangan / pesan."

#: ../satellite_tools/satsync.py:2326
#, fuzzy
msgid "  2  - normal level of logging/messaging."
msgstr "2 - tingkat normal penebangan / pesan."

#: ../satellite_tools/satsync.py:2327
#, fuzzy
msgid "  3  - lots of logging/messaging."
msgstr "3 - banyak penebangan / pesan."

#: ../satellite_tools/satsync.py:2328
#, fuzzy
msgid "  4+ - excessive logging/messaging."
msgstr "4+ - penebangan/pesan yang berlebihan."

#: ../satellite_tools/satsync.py:2341
#, fuzzy
msgid "ERROR: The --master option is only valid with the --mount-point option"
msgstr "GALAT: Opsi --master hanya berlaku dengan opsi --mount-point"

#: ../satellite_tools/satsync.py:2351
#, fuzzy, python-format
msgid "ERROR: Unable to lookup Org Id %s"
msgstr "GALAT: Tidak dapat mencari Id Organisasi %s"

#: ../satellite_tools/satsync.py:2360
#, fuzzy
msgid ""
"WARNING: --list-channels option overrides any --step option. --step ignored."
msgstr ""
"PERINGATAN: Opsi --list-channels menimpa opsi --step apa pun. --langkah "
"diabaikan."

#: ../satellite_tools/satsync.py:2375
#, fuzzy, python-format
msgid ""
"ERROR: '%s' is not a valid step. See 'man mgr-inter-sync' for more detail."
msgstr ""
"GALAT: '%s' bukan langkah yang valid. Lihat 'man satellite-sync' untuk "
"detail lebih lanjut."

#: ../satellite_tools/satsync.py:2407
#, fuzzy
msgid ""
"ERROR: No channels currently imported; try mgr-inter-sync --list-channels; "
"then mgr-inter-sync -c chn0 -c chn1..."
msgstr ""
"GALAT: Tidak ada saluran yang saat ini diimpor; coba mgr-inter-sync --list-"
"channels; kemudian mgr-inter-sync -c chn0 -c chn1..."

#. int(None) --> TypeError
#. int('a')  --> ValueError
#: ../satellite_tools/satsync.py:2457 ../satellite_tools/satsync.py:2461
#, fuzzy
msgid "ERROR: --batch-size must have a value within the range: 1..50"
msgstr "GALAT: --batch-size harus memiliki nilai dalam kisaran: 1..50"

#: ../satellite_tools/satsync.py:2469
#, fuzzy, python-format
msgid "ERROR: no such directory %s"
msgstr "Galat sewaktu membuat direktori %s."

#: ../satellite_tools/satsync.py:2474
#, fuzzy
msgid "Error Codes: Returned codes means:"
msgstr "Kode kesalahan: Kode yang dikembalikan berarti:"

#: ../satellite_tools/satsync.py:2475
#, fuzzy
msgid " -1  - Could not lock file or KeyboardInterrupt or SystemExit"
msgstr "-1 - Tidak dapat mengunci file atau KeyboardInterrupt atau SystemExit"

#: ../satellite_tools/satsync.py:2476
#, fuzzy
msgid "  0  - User interrupted or intentional exit"
msgstr "0 - Pengguna terganggu atau disengaja keluar"

#: ../satellite_tools/satsync.py:2477
#, fuzzy
msgid "  1  - attempting to run more than one instance of mgr-inter-sync."
msgstr "1 - mencoba untuk menjalankan lebih dari satu contoh mgr-inter-sync."

#: ../satellite_tools/satsync.py:2478
#, fuzzy
msgid "  2  - Unable to find synchronization tools."
msgstr "2 - Tidak dapat menemukan alat sinkronisasi."

#: ../satellite_tools/satsync.py:2479
#, fuzzy
msgid "  3  - a general socket exception occurred"
msgstr "3 - pengecualian soket umum terjadi"

#: ../satellite_tools/satsync.py:2480
#, fuzzy
msgid "  4  - an SSL error occurred. Recheck your SSL settings."
msgstr "4 - terjadi kesalahan SSL. Periksa kembali pengaturan SSL Anda."

#: ../satellite_tools/satsync.py:2481
#, fuzzy
msgid "  5  - ISS error"
msgstr "5 - Kesalahan ISS"

#: ../satellite_tools/satsync.py:2482
#, fuzzy
msgid "  6  - unhandled exception occurred"
msgstr "6 - pengecualian ditangani terjadi"

#: ../satellite_tools/satsync.py:2483
#, fuzzy
msgid "  7  - unknown sync error"
msgstr "7 - kesalahan sinkronisasi tidak diketahui"

#: ../satellite_tools/satsync.py:2484
#, fuzzy
msgid "  8  - ERROR: must be root to execute"
msgstr "8 - ERROR: harus root untuk mengeksekusi"

#: ../satellite_tools/satsync.py:2485
#, fuzzy
msgid "  9  - rpclib fault during synchronization init"
msgstr "9 - rpclib kesalahan selama sinkronisasi init"

#: ../satellite_tools/satsync.py:2486
#, fuzzy
msgid "  10 - synchronization init error"
msgstr "10 - sinkronisasi init kesalahan"

#: ../satellite_tools/satsync.py:2487
#, fuzzy
msgid "  11 - Error parsing XML stream"
msgstr "11 - Kesalahan parsing XML stream"

#: ../satellite_tools/satsync.py:2488
#, fuzzy
msgid "  12 - Channel do not exist"
msgstr "12 - Saluran tidak ada"

#: ../satellite_tools/satsync.py:2489
#, fuzzy
msgid "  13 - SQL error during importing package metadata"
msgstr "13 - Galat SQL selama mengimpor metadata paket"

#: ../satellite_tools/satsync.py:2490
#, fuzzy
msgid "  14 - SQL error during linking channel packages"
msgstr "14 - Galat SQL selama menghubungkan paket saluran"

#: ../satellite_tools/satsync.py:2491
#, fuzzy
msgid "  15 - SQL error during xml processing"
msgstr "15 - Galat SQL selama pemrosesan xml"

#: ../satellite_tools/satsync.py:2492
#, fuzzy
msgid "  16 - server.mount_point not set in the configuration file"
msgstr "16 - server.mount_point tidak ditetapkan dalam file konfigurasi"

#: ../satellite_tools/satsync.py:2493
#, fuzzy
msgid ""
"  17 - SQL error during retrieving the channels already imported in the SUSE "
"Manager database"
msgstr ""
"17 - GALAT SQL selama mengambil saluran yang sudah diimpor dalam database "
"SUSE Manager"

#: ../satellite_tools/satsync.py:2494
#, fuzzy
msgid "  18 - Wrong db connection string in rhn.conf"
msgstr "18 - String koneksi db salah di rhn.conf"

#: ../satellite_tools/satsync.py:2495
#, fuzzy
msgid "  19 - Bad arguments"
msgstr "19 - Argumen buruk"

#: ../satellite_tools/satsync.py:2496
#, fuzzy
msgid "  20 - Could not connect to db."
msgstr "20 - Tidak dapat terhubung ke db."

#: ../satellite_tools/satsync.py:2497
#, fuzzy
msgid "  21 - Bad debug level"
msgstr "21 - Tingkat debug buruk"

#: ../satellite_tools/satsync.py:2498
#, fuzzy
msgid "  22 - Not valid step"
msgstr "22 - Tidak berlaku langkah"

#: ../satellite_tools/satsync.py:2499
#, fuzzy
msgid "  24 - no such file"
msgstr "24 - tidak ada file tersebut"

#: ../satellite_tools/satsync.py:2500
#, fuzzy
msgid "  25 - no such directory"
msgstr "25 - tidak ada direktori tersebut"

#: ../satellite_tools/satsync.py:2501
#, fuzzy
msgid "  26 - mount_point does not exist"
msgstr "26 - mount_point tidak ada"

#: ../satellite_tools/satsync.py:2502
#, fuzzy
msgid "  27 - No such org"
msgstr "27 - Tidak ada organisasi seperti itu"

#: ../satellite_tools/satsync.py:2503
#, fuzzy
msgid "  28 - error: --master is only valid with --mount-point"
msgstr "28 - error: --master hanya berlaku dengan --mount-point"

#: ../satellite_tools/satsync.py:2510
#, fuzzy, python-format
msgid "ERROR: unknown dump version, try one of %s"
msgstr "GALAT: versi dump tidak diketahui, coba salah satu dari %s"

#, fuzzy
#~ msgid "Red Hat Satellite Welcome Message"
#~ msgstr "Red Hat Satelit Selamat Datang Pesan"
